tests/component/test_trigger_properties.py

```diff
@@ -0,0 +1,136 @@
+"""Contains a collection of pytest tests that validates trigger properties."""
+
+import nidaqmx
+from nidaqmx.constants import TriggerType
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+
+
+def test__trigger__get_float_property__returns_value(any_x_series_device):
+    """Test to validate getter for trigger property of float type."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
```

Comment:
#241 added a fixture for channels, which I like. There is one issue with their impl that I believe is resolvable, but I think its a good idea to do the same for these tests to remove a bunch of duplicate code!

Reply:
Added fixture for this block of code.